mp/src/vgui2/vgui_controls/Button.cpp

```diff
@@ -695,12 +695,12 @@ void Button::SetMouseClickEnabled(MouseCode code,bool state)
 	if(state)
 	{
 		//set bit to 1
-		_mouseClickMask|=1<<((int)(code+1));
```

Comment:
I don't know why they did that `+1` here. In `MouseButtonBit()` I use bit #0, which they may have wanted to avoid for some reason.

Reply:
Who knows, VGUI is a horrible poorly documented mess. If that resolves it I'm happy but should probably double check for regressions elsewhere before merge.